integration-tests/js-compute/fixtures/extend-from-builtins/extend-from-builtins.ts

```diff
@@ -27,8 +30,9 @@ const builtins = [
               super("gzip")
               break;
             }
-            case "Dictionary": {
-              super("example")
+            case "Dictionary":
+            case "ObjectStore": {
+              super("hello")
```

Comment:
This would actually throw an error due to no ObjectStore named 'hello' existing and being linked to this application. When Viceroy has ObjectStore implemented, we can get this test to pass